src/frontend-scripts/components/Gamenotes.jsx

```diff
@@ -65,6 +69,7 @@ class Gamenotes extends React.Component {
 	render() {
 		const notesChange = e => {
 			this.props.changeNotesValue(`${e.target.value}`);
+			localStorage.setItem('GameNotes', e.target.value);
```

Comment:
This may pose issues down the line with setting the entire string into localstorage, as the text body gets larger and larger - given that LS reads/writes are synchronous and this would potentially end up stalling the page. But we can see down the line if it actually has any noticeable implications/what size of notes would really start to cause an issue. Odds are they'd hit the 5MB LS limit before actually causing user-noticeable performance implications, but I figured I'd bring it up anyways. If it does become an issue we can look into either debouncing the LS saves, or swapping to use localForage instead (which is async, and will bypass the 5MB limit).

Reply:
I did consider that and I tried some small spam with my performance monitor on with no noticible performance hit